gengo/gengo.py

```diff
@@ -418,6 +396,54 @@ def replaceURLAttachmentsWithAttachments(self, obj):
             obj['attachments'] = obj['url_attachments']
             del obj['url_attachments']
 
+    def _raiseForMultipleErrorResponse(self, results):
+        error_codes = []
+        messages = []
+        for job_key, msg_code_list in results['err'].items():
+            messages.append(
+                '<{0}: {1}>'.format(job_key, msg_code_list[0]['msg']))
+            error_codes.append(msg_code_list[0]['code'])
+            if not self.debug:
+                continue
+
+            for message_code in msg_code_list:
+                logger.error(
+                    "Gengo returned an error, code: %s, message: %s",
+                    message_code['code'], message_code['msg']
+                )
+
+        concatted_msg = ' '.join(messages)
```

Comment:
[really not important]
small note on the word :
```diff
- concatted
+ concatenated
```

it's not a big issue, really!

alternatively, we can, perhaps, just use `msg = ' '.join(messages)`?